docs/progress-bars.md

```diff
@@ -1,37 +1,11 @@
 # Progress bars
 
-!!! warning
-    This page will soon replace <https://docs.easybuild.io/en/latest/Progress_bars.html>.
+(supported since: EasyBuild v4.5.0)
```

Comment:
use `*...*` to make it italic?

```suggestion
*(supported since: EasyBuild v4.5.0)*

Reply:
See https://github.com/easybuilders/easybuild-docs/pull/10#issue-1424453640

Reply:
Oh, I overlooked that... Ugh, that's a bit... weird.

Would it make sense to disable `MD036` in the linter?

Reply:
I think so, as we have a legitimate reason to allow it. Done in 2de6427